README.md

```diff
@@ -168,6 +168,10 @@ cause the user user to be prompted for all the required data in the command prom
 vcfg will use that data to generate a rabbitmq_config.yml file in the `VOLTTRON_HOME` 
 directory.
 
+If the above configuration file is being used as a basis, be sure to update it with 
+the hostname of the deployment (this should be the fully qualified domain name
```

Comment:
What if I am running it locally, and don't have a domain name that I'm using? As a beginner, how do I find the fully qualified domain name, or generate one?